org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/formatter/IValueFormatter.java

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+* Copyright (c) 2017 Microsoft Corporation and others.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v1.0
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v10.html
+*
+* Contributors:
+*     Microsoft Corporation - initial API and implementation
+*******************************************************************************/
+
+package org.eclipse.jdt.ls.debug.adapter.formatter;
+
+import com.sun.jdi.Type;
+import com.sun.jdi.Value;
+
+import java.util.Map;
+
+public interface IValueFormatter extends IFormatter {
+    /**
+     * The conditional function for this formatter.
+     *
+     * @param type the JDI type
+     * @return whether or not this formatter is expected to work on this value.
+     */
+    boolean acceptType(Type type, Map<String, Object>props);
```

Comment:
This has the same signature with ITypeFormatter, can it be placed in the IFormatter interface?